molecule.yml

```diff
@@ -1,9 +1,14 @@
 ---
 ansible:
   config_file: ansible.cfg
-  requirements_file: requirements.yml
   verbose:
-  sudo: True
+  become: True
+  host_vars:
+    ansible-jenkins-xenial64:
+      ansible_python_interpreter: /usr/bin/python3
```

Comment:
Is this needed because Xenial doesn't symlink `python` -> `python3`, or something like that, by default?

Reply:
That's right, yeah.